js/models/event_handler.js

```diff
@@ -138,6 +147,15 @@ export default class EventHandler {
       this.model.save_changes();
     });
 
+    /* Rotation control */
+    this.model.on("change:_rotation", () => {
+      setRotation(this.model.get("_rotation"), this.aladinDiv);
```

Comment:
question: I'm unsure where this `setRotation` method is coming from, should this be `this.aladin.setRotation`?